net-dialup/freeradius-client/freeradius-client-1.1.7.ebuild

```diff
@@ -1,12 +1,12 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="5"
+EAPI=7
```

Comment:
Please revbump with this change, and reset keywords to ~.
`ekeyword ~all freeradius-client-1.1.7.ebuild`

Reply:
Doing